pkg/kubectl/describe.go

```diff
@@ -438,6 +441,7 @@ func (d *PersistentVolumeDescriber) Describe(namespace, name string) (string, er
 
 	return tabbedString(func(out io.Writer) error {
 		fmt.Fprintf(out, "Name:\t%s\n", pv.Name)
+		fmt.Fprintf(out, "Namespace:\t%s\n", pv.Namespace)
```

Comment:
This is a non-namespaced resource.

See: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/api/latest/latest.go#L95

Remove this line.